tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -586,7 +586,7 @@ Full list of hints below:
 +================+=============================+=====================================+
 | **vec4**       | hint_color                  | Used as color                       |
 +----------------+-----------------------------+-------------------------------------+
-| **int, float** | hint_range(min,max [,step]) | Used as range (with min/max/step)   |
+| **int, float** | hint_range(min, max[, step]) | Used as range (with min/max/step)   |
```

Comment:
I think this one breaks the table layout and the table needs to be adjusted? :)

Reply:
Fixed.